scripts/ms

```diff
@@ -35,6 +35,7 @@ def parse(args):
     parser.add_argument('config')
     parser.add_argument('-n', '--n-processes', default=1, type=int)
     parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('-a', '--annual', action='store_true')
```

Comment:
did we talk about moving this to the config file where the option would be something like `grouper` and the user could specify something like `time.year` or `time.month`?